src/plone/api/content.py

```diff
@@ -473,3 +474,45 @@ def get_uuid(obj=None):
     :Example: :ref:`content_get_uuid_example`
     """
     return IUUID(obj)
+
+
+def find(context=None, depth=None, **kwargs):
+    """Find content
+
+    :param context: Context within we want to search
```

Comment:
This sentence should read "context within which we want to search" or simply "context for the search"